rama-net/Cargo.toml

```diff
@@ -18,10 +18,11 @@ default = []
 http = ["dep:rama-http-types", "dep:sha2", "dep:itertools", "dep:hex"]
 tls = ["dep:hex", "dep:md5", "dep:sha2", "dep:itertools"]
 rustls = ["tls", "dep:rustls"]
-boring = ["tls", "dep:boring", "dep:nom"]
+boring = ["tls", "dep:boring", "dep:nom", "dep:aws-lc-sys"]
```

Comment:
Not for boring, but for `rustls`, boring doesn't use this crypto library it has its own crypto